SINGLE-VALIDATION.md

```diff
@@ -0,0 +1,60 @@
+# Validation - Single Safe
```

Comment:
You'll need to add in CI checks. See here: https://github.com/ethereum-optimism/superchain-ops/pull/848/commits/c56f26c42b623faaa898f17a0b7180d4ffafb09f

Reply:
Given this is single, it'll not be the nested commands I linked.

Reply:
a76799659645